web-server/src/components/TeamsList.tsx

```diff
@@ -54,6 +56,13 @@ export const TeamsList = () => {
     (state) => state.team?.requests?.teams === FetchState.REQUEST
   );
 
+  useEffect(() => {
+    if (router.query.create === 'true') {
+      showCreate.true();
+      router.replace(router.pathname, '');
+    }
+  }, [router, router.query.create, showCreate]);
```

Comment:
see, you only need to `showCreate.true` in your dependency array, but my guess is linter asked you to add the entire `showCreate` object.

- instead of calling `showCreate.true()`, use `depFn(showCreate, true)`,
- update the dependency array to include `showCreate.true`, instead of `showCreate`

Reply:
https://middle-out-group.slack.com/archives/C06U2M1UEA0/p1724305199044449

Reply:
Understood